__tests__/indoor-directions-steps.test.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+
+import IndoorDirectionSteps from "../src/components/travel-route/indoor-directions-steps.component";
+import PathFindingService from "../src/services/pathfinding.service";
+import renderer from "react-test-renderer";
+import { shallow } from "enzyme";
+import * as testData from "./__testData__/indoor-travel-route";
+
+const mockFloorPath = PathFindingService.getInstance().findPathBetweenPOIs(
+  testData.mockPOIs[0],
+  testData.mockPOIs[1]
+);
```

Comment:
findPathBetweenPOIs takes accessibilityMode: boolean, does that default to false?